src/pages/edge/edge-network/tabs/api-reference.md

```diff
@@ -516,57 +801,88 @@ public final class ExperienceEvent {
-      *
-      * @param xdm {@code Map<String, Object>} of raw XDM schema data
-      * @param datasetIdentifier The Experience Platform dataset identifier where this event is sent.
-      *                          If not provided, the default dataset defined in the configuration ID is used
-      * @return instance of current builder
-      * @throws UnsupportedOperationException if this instance was already built
-      */
+    * Solution specific XDM event data and dataset identifier for this event.
+    *
+    * @param xdm {@code Map<String, Object>} of raw XDM schema data
+    * @param datasetIdentifier The Experience Platform dataset identifier where this event is sent.
+    *                          If not provided, the default dataset defined in the configuration ID is used
+    * @return instance of current builder
+    * @throws UnsupportedOperationException if this instance was already built
+    */
     public Builder setXdmSchema(final Map<String, Object> xdm, final String datasetIdentifier) {...}
 
     /**
-      * Solution specific XDM event data for this event, passed as raw mapping of keys and
-      * Object values.
-      *
-      * @param xdm {@code Map<String, Object>} of raw XDM schema data
-      * @return instance of current builder
-      * @throws UnsupportedOperationException if this instance was already built
-      */
+    * Solution specific XDM event data for this event, passed as raw mapping of keys and
+    * Object values.
+    *
+    * @param xdm {@code Map<String, Object>} of raw XDM schema data
+    * @return instance of current builder
+    * @throws UnsupportedOperationException if this instance was already built
+    */
     public Builder setXdmSchema(final Map<String, Object> xdm) {...}
 
     /**
-      * Builds and returns a new instance of {@code ExperienceEvent}.
-      *
-      * @return a new instance of {@code ExperienceEvent} or null if one of the required parameters is missing
-      * @throws UnsupportedOperationException if this instance was already built
-      */
+    * Override the default datastream identifier to send this event's data to a different datastream.
+    *
+    * When using {@link Edge#sendEvent}, this event is sent to the Experience Platform using the
+    * datastream identifier {@code datastreamIdOverride} instead of the default Experience Edge
+    * configuration ID set in the SDK Configuration key {@code edge.configId}.
+    *
+    * @param datastreamIdOverride Datastream identifier to override the default datastream identifier set in the Edge configuration
+    * @return instance of current builder
+    * @throws UnsupportedOperationException if this instance was already built
+    *
+    */
+    public Builder setDatastreamIdOverride(final String datastreamIdOverride) {...}
+
+    /**
+    * Override the default datastream configuration to trigger different datastream behaviors than the
+    * default ones for this event.
```

Comment:
Override the default datastream configuration settings for individual services for this event.